plutus-scb/app/Main.hs

```diff
@@ -89,6 +96,8 @@ commandParser =
              [ migrationParser
              , allServersParser
              , mockWalletParser
+             , scbWebserverParser
+             , psGeneratorCommandParser
              , mockNodeParser
              , chainIndexParser
              , signingProcessParser
```

Comment:
whay are these called `xParser` and not `xCommand`?

Reply:
Because they aren't commands, they're parsers of commands.

Reply:
Because they aren't commands, they're parsers?